testing/chain.go

```diff
@@ -457,7 +464,12 @@ func (chain *TestChain) CreateTMClientHeader(chainID string, blockHeight int64,
 	blockID := MakeBlockID(hhash, 3, tmhash.Sum([]byte("part_set")))
 	voteSet := tmtypes.NewVoteSet(chainID, blockHeight, 1, tmproto.PrecommitType, tmValSet)
 
-	commit, err := tmtypes.MakeCommit(blockID, blockHeight, 1, voteSet, signers, timestamp)
+	var signerArr []tmtypes.PrivValidator
+	for _, v := range tmValSet.Validators {
+		signerArr = append(signerArr, signers[v.Address.String()])
+	}
+
+	commit, err := tmtypes.MakeCommit(blockID, blockHeight, 1, voteSet, signerArr, timestamp)
```

Comment:
Does MakeCommit sort the signerArr?

Reply:
No, the signer arr is passed in with the order of the voteset. see above for loop